chellow/templates/sites.html

```diff
@@ -18,8 +18,8 @@
 		<div class="section-data">
 			<div class="standardForm textWidthAuto">
 				<div class="row">
-					<div class="col-sm-6">	
-						<form>
+					<div class="col-sm-6">
+						<form action="">
 							<fieldset>
```

Comment:
The 'action' attribute shouldn't be empty. See the HTML5 spec:

https://www.w3.org/TR/html5/forms.html#attr-fs-action